frontend/public/components/filter-toolbar.tsx

```diff
@@ -234,6 +235,14 @@ const FilterToolbar_: React.FC<FilterToolbarProps & RouteComponentProps> = (prop
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  // Default filters
```

Comment:
Instead of creating this new hook, I think you can modify the hook immediately above to use `defaultSelected` when `selectedRowFilters` is empty.

In other words, replace
```
!_.isEmpty(selectedRowFilters) && applyRowFilter(selectedRowFilters);
```
with
```
    if (_.isEmpty(selectedRowFilters)) {
      rowFilters.forEach((filter) => updateRowFilterSelected(filter.defaultSelected));
    } else {
      applyRowFilter(selectedRowFilters);
    }
```